source/docs/guides/merchant_report.md

```diff
@@ -61,7 +61,6 @@ summarized in the table below.
 | `recurring_billing_enabled` | Does the merchant have recurring billing enabled |
 | `credit_card_batch_enabled` | Does the merchant have credit card batch enabled |
 | `eft/ach_batch_enabled` | Does the merchant have eft/ach batch enabled |
-| `interac_online_enabled` | Does the merchant have interac online enabled |
```

Comment:
We are going to need to leave this field in since removing a field could break a merchant integration

Reply:
Maybe change the text here would be good:

The Interac Online feature has been decommissioned.  The field is included in the response for backwards compatibility.

Reply:
updated